shyftBlockExplorerApi/handler.go

```diff
@@ -195,25 +195,37 @@ func GetRecentBlock(w http.ResponseWriter, r *http.Request) {
 }
 
 //GetInternalTransactions gets internal txs
-func GetInternalTransactions(w http.ResponseWriter, r *http.Request) {
+func GetInternalTransactionsByHash(w http.ResponseWriter, r *http.Request) {
+	sqldb, err := core.DBConnection()
+
 	vars := mux.Vars(r)
-	address := vars["address"]
+	txHash := vars["txHash"]
+
+	internalTxs := core.SGetInternalTransaction(sqldb, txHash)
+	if err != nil {
+		http.Error(w, err.Error(), 500)
+		return
+	}
 
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 	w.WriteHeader(http.StatusOK)
 
-	fmt.Fprintln(w, "Get InternalTransactions", address)
+	fmt.Fprintln(w, internalTxs)
 }
```

Comment:
Is this function doing anything? It seems like it's not sending the internal tx back

Reply:
I dont understand? Why dont you think the getter functions and the API handler functions are sending the internals txs back?

Reply:
Nevermind i set it up incorrectly.